ticdc/ticdc-compatibility.md

```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
 
 本文介绍了与 TiCDC 有关的一系列兼容性问题及其处理方案。
 
-<!--
-## 组件兼容性矩阵
+## TiCDC 和其他 TiDB 组件的兼容性
```

Comment:
```suggestion
```

Reply:
Simplify the heading structure by removing the redundant parent heading, as there is only one compatibility section specifically for TiDB Lightning.